medusa/providers/torrent/rss/nyaa.py

```diff
@@ -48,15 +48,21 @@ def search(self, search_strings, age=0, ep_obj=None, **kwargs):
 
         :param search_strings: A dict with mode (key) and the search value (value)
         :param age: Not used
-        :param ep_obj: Not used
+        :param ep_obj: An episode object
         :returns: A list of search results (structure)
         """
         results = []
 
         # Search Params
+        if ep_obj:
```

Comment:
I think this can be a bit more DRY like:
```
category = '1_0'
if ep_obj and ep_obj.series.is_anime:
    category = '4_0'
```

Reply:
Indeed, it makes more sense, but it'd be something more like:

```
category = '1_0'
if ep_obj and not ep_obj.series.is_anime:
    category = '4_0'
```

Since 4_0 is Live Action :)